jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/resources/org/jbpm/workbench/cm/client/resources/i18n/Constants.properties

```diff
@@ -89,3 +89,6 @@ ActionStart=Start
 StartedAt=Started at
 Dynamic=Dynamic
 AvailableIn=Available in
+Yes=Yes
```

Comment:
I think the label for the button should be _Delete_ and use _btn-danger_ to highlight.